src/dos/dos_keyboard_layout.cpp

```diff
@@ -1065,6 +1066,9 @@ KeyboardErrorCode KeyboardLayout::ReadCodePageFile(const char *requested_cp_file
 			}
 			INT10_SetupRomMemoryChecksum();
 
+			// adapt UTF-8 [autoexec] section to new code page
```

Comment:
Would convert be a better word? Because that's what we're gonna do, right, a codepage conversion?

Reply:
OK, changed. Actually, it converts from UTF-8 to current code page. Converting between code pages is never done - since converting from UTF-8 is potentially lossy operation (we might need to drop some accents, etc.), we always convert from the UTF-8 original.